content/tutorials/marketplace-fees.md

```diff
@@ -0,0 +1,83 @@
+
+## Fee types
+
+### Swap fee
+
+Swap fees are collected whenever someone swaps a datatoken for base token (e.g., OCEAN) or base token for datatoken. The swap could be inside a pool (using an automated market maker) or in a fixed-rate exchange.
+These are the fees that are applied whenever a user swaps base token or datatoken:
+
+- Publisher Marketplace swap fee
+- Consumer Marketplace swap fee
+- Provider Consumption Fees
+- [Ocean Community Fee](#ocean-community-fee)
+
+### Publish fee
+
+Publish fees will be charged to a publisher when they publish an asset.
+
+Currently, no fees are applicable in Ocean's default marketplace. However, it is possible to charge a publishing fee in the custom marketplace by adding an extra transaction in the publish flow.
+
+Based on the use case of the marketplace, the marketplace owner can decide if this fee should be charged or not.
+
+### Consume fee
+
+Consume fee is charged when a user holding datatoken exchanges it for a compute service, downloading a data asset or an algorithm.
+
+The following fees apply to the consumption of data assets as a part of Consume fee:
+
+- Consume Market Consumption Fee
+- Publisher Market Consumption Fee
+- Provider Consumption Fees
+- [Ocean Community Fee](#ocean-community-fee)
+
+### Ocean Community fee
+
+Ocean's smart contract collects the **Ocean Community fee** during a swap operation. These fees are reinvested in community projects via Ocean DAO and other initiatives. Currently, this fee is set to 0.1% (as long as either Ocean or H20 are used as the base token in the market) in the smart contract but can be updated by the Ocean Protocol foundation.
+
+Whereas a fee of 0.2% is applicable on all swaps using another token as the base token.
+
+### Provider fee
+
+Provider is a component of Ocean Protocol's ecosystem which facilitates data consumption, starts compute jobs, encrypts DDOs, and decrypts DDOs. Provider also validates if the user can access a particular data asset or service. To know more about Provider, click [here](https://github.com/oceanprotocol/provider).
+
+Provider fees are paid to the individual or organization running their Provider instance whenever a user consumes a data asset or swaps a datatoken for a base token. These fees can be set to an absolute amount, and the receiver can also decide which token to receive the fees in - they don't have to be in the same currency used in the consuming market.
+
+Provider fee can also be used to charge for computing resources. Based on the compute resources needed to run an algorithm in the Compute-to-Data environment, a consumer can choose the amount to pay according to their needs.
+
+This fee incentivizes individuals and organizations to run their provider instances and charge consumers according to resource usage.
```

Comment:
Consider making fees plural here - to match the convention used throughout the other sections.